pkg/backporter/backporter.go

```diff
@@ -184,6 +197,32 @@ const clonesTemplateConstructor = `
     </form>
 </div>`
 
+const helpTemplateConstructor = `
+<h2 id="title"><a href="#title">Why a bugzilla-backporter?</a></h2>
+
+<p>
+The <code>Bugzilla Backporter</code> would allow the user to easily understand how different clones
+are related to each other and which release they are targeted for.
+It also allows the user to create clones targeting a specific release.
+</p>
+<h2 id="title"><a href="#title">How to find clones?</a></h2>
+
+<p>
+The user would enter a bugID for the bug whose clones need to be found in the input field on the top right 
```

Comment:
Maybe use imperative consistenly and tell the user what to do? "_Enter the bug ID into the input field (...) The tool will show a list of all clones..._ etc.